CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## [Unreleased]
+- Stopped 2.11 release as there are multiple incompatibilities.
```

Comment:
Try to follow https://keepachangelog.com/en/1.0.0/ here, especially the `Types of changes` section which has gone missing in recent updates to this file.

(We also leave a blank line after headings)